<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>KeypadNew</class>
 <widget class="QDialog" name="KeypadNew">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>332</width>
    <height>433</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Enter Value</string>
  </property>
  <layout class="QGridLayout" name="gridLayout" rowstretch="0,1,1,1,1,0" columnstretch="1,1,1,1">
   <item row="1" column="0" colspan="4">
    <widget class="QLineEdit" name="valueLine">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>74</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Andale Mono</family>
       <pointsize>22</pointsize>
      </font>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QPushButton" name="_7Button">
     <property name="minimumSize">
      <size>
       <width>74</width>
       <height>74</height>
      </size>
     </property>
     <property name="text">
      <string>7</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QPushButton" name="_8Button">
     <property name="minimumSize">
      <size>
       <width>74</width>
       <height>74</height>
      </size>
     </property>
     <property name="text">
      <string>8</string>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QPushButton" name="_9Button">
     <property name="minimumSize">
      <size>
       <width>74</width>
       <height>74</height>
      </size>
     </property>
     <property name="text">
      <string>9</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QPushButton" name="_4Button">
     <property name="minimumSize">
      <size>
       <width>74</width>
       <height>74</height>
      </size>
     </property>
     <property name="text">
      <string>4</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QPushButton" name="_5Button">
     <property name="minimumSize">
      <size>
       <width>74</width>
       <height>74</height>
      </size>
     </property>
     <property name="text">
      <string>5</string>
     </property>
    </widget>
   </item>
   <item row="3" column="2">
    <widget class="QPushButton" name="_6Button">
     <property name="minimumSize">
      <size>
       <width>74</width>
       <height>74</height>
      </size>
     </property>
     <property name="text">
      <string>6</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QPushButton" name="_1Button">
     <property name="minimumSize">
      <size>
       <width>74</width>
       <height>74</height>
      </size>
     </property>
     <property name="text">
      <string>1</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QPushButton" name="_2Button">
     <property name="minimumSize">
      <size>
       <width>74</width>
       <height>74</height>
      </size>
     </property>
     <property name="text">
      <string>2</string>
     </property>
    </widget>
   </item>
   <item row="4" column="2">
    <widget class="QPushButton" name="_3Button">
     <property name="minimumSize">
      <size>
       <width>74</width>
       <height>74</height>
      </size>
     </property>
     <property name="text">
      <string>3</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QPushButton" name="_0Button">
     <property name="minimumSize">
      <size>
       <width>74</width>
       <height>74</height>
      </size>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QPushButton" name="plusButton">
     <property name="minimumSize">
      <size>
       <width>74</width>
       <height>74</height>
      </size>
     </property>
     <property name="text">
      <string>+</string>
     </property>
    </widget>
   </item>
   <item row="5" column="2">
    <widget class="QPushButton" name="minusButton">
     <property name="minimumSize">
      <size>
       <width>74</width>
       <height>74</height>
      </size>
     </property>
     <property name="text">
      <string>-</string>
     </property>
    </widget>
   </item>
   <item row="3" column="3">
    <widget class="QPushButton" name="clearButton">
     <property name="minimumSize">
      <size>
       <width>74</width>
       <height>74</height>
      </size>
     </property>
     <property name="text">
      <string>Clear</string>
     </property>
    </widget>
   </item>
   <item row="2" column="3">
    <widget class="QPushButton" name="backButton">
     <property name="minimumSize">
      <size>
       <width>74</width>
       <height>74</height>
      </size>
     </property>
     <property name="text">
      <string>&lt;-</string>
     </property>
    </widget>
   </item>
   <item row="5" column="3">
    <widget class="QPushButton" name="cancelButton">
     <property name="minimumSize">
      <size>
       <width>74</width>
       <height>74</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="jle_resources.qrc">
       <normaloff>:/icons/Resources/pic/Button_NOK.png</normaloff>:/icons/Resources/pic/Button_NOK.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>64</width>
       <height>64</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="4">
    <widget class="QLabel" name="titleLabel">
     <property name="sizePolicy">
      <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="styleSheet">
      <string notr="true">background: rgb(255,255,255)</string>
     </property>
     <property name="text">
      <string>Enter Value</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="jle_resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>cancelButton</sender>
   <signal>clicked()</signal>
   <receiver>KeypadNew</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>77</x>
     <y>467</y>
    </hint>
    <hint type="destinationlabel">
     <x>271</x>
     <y>432</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
